### PR TITLE
Add a 96+ VMs job

### PR DESCRIPTION
**Pull request by kbaikov**
This is job to test 100+ nodes with Crowbar in virtualized test environment.
Currently reserved to be run on uv300x.arch.suse.de machine.

### CODE REVIEW

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            ### Admin Node Configuration ###
+            admin_node_memory=31457280
+            adminvcpus=10
+            adminnode_hdd_size=100 # GB
+            ### Compute Node Configuration ###
+            compute_node_memory=2097152 # KB's
+            vcpus=1 # Default 1 vcpu
+            TESTHEAD=1
+            cloudsource=develcloud{version}
+            nodenumber=96
```
Comment (aplanas) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml]: so is not 100+? maybe fix the commit message?

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+- job-template:
+    name: 'cloud-mkcloud{version}-job-96compute-{arch}'
+    node: cloud-trigger
+
+    triggers:
+      - timed: '0 0 * * 6'
```
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml]: why do you time it to a specific time? just use H..

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            clusterconfig="data+services=3:network=3"
+            want_node_aliases='controller=3,network=3,compute=90'
+            drbd_hdd_size=100
+            hacloud=1
+            networkingplugin=openvswitch
+            networkingmode=vlan
+            want_dvr=1
+            cephvolumenumber=0
+            mkcloudtarget=all
+            label=big_machine
```
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml]: that should probably be  a architecture specific label, e.g. label={label} with label being expanded above in the cloud-mkcloud7.yaml

File: scripts/jenkins/jobs-ibs/cloud-mkcloud7.yaml [hunk 4]
```diff
@@ -13,6 +13,7 @@
     label: openstack-mkcloud-SLE12-x86_64
     sles12sp2: 0
     jobs:
+        - 'cloud-mkcloud{version}-job-96compute-{arch}'
```
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/cloud-mkcloud7.yaml]: as that job can no be run on the openstack-mkcloud-SLE12-x86_64 workers, you need to add a separate section for running the job on a "openstack-mkcloud-SLE12-bigmachine-x86_64" (or something like that).

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+    builders:
+      - trigger-builds:
+        - project: openstack-mkcloud
+          condition: SUCCESS
+          block: true
+          current-parameters: true
+          predefined-parameters: |
+            ### Controller Node Configuration ###
+            controller_node_memory=31457280
```
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml]: why lower than default memory?

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+            controller_hdd_size=20
+            ### Admin Node Configuration ###
+            admin_node_memory=31457280
```
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml]: why lower than default memory?

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    builders:
+      - trigger-builds:
+        - project: openstack-mkcloud
+          condition: SUCCESS
+          block: true
+          current-parameters: true
+          predefined-parameters: |
+            ### Controller Node Configuration ###
+            controller_node_memory=31457280
+            controller_hdd_size=20
```
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml]: thats default, remove it

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+            ### Admin Node Configuration ###
+            admin_node_memory=31457280
+            adminvcpus=10
+            adminnode_hdd_size=100 # GB
+            ### Compute Node Configuration ###
+            compute_node_memory=2097152 # KB's
```
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml]: lower than default?

File: scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml [hunk 9]
```diff
@@ -0,0 +1,43 @@
+- job-template:
+    name: 'cloud-mkcloud{version}-job-96compute-{arch}'
```
Comment (dirkmueller) [inline scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-96compute-template.yaml]: probably should be 90compute then not 96compute..

File: scripts/jenkins/jobs-ibs/cloud-mkcloud7.yaml (hunk 4 above)
Comment (kbaikov) [inline scripts/jenkins/jobs-ibs/cloud-mkcloud7.yaml]: Done. Label expanded. Also updated jenkins node label: https://ci.suse.de/label/openstack-mkcloud-SLE12-big_machine-x86_64/

**Comment by kbaikov** [comment]
I am a bit hesitant to increase  the memory, since the this machine has only 256 GB of ram.
Also only 64 vCPUs.

**Comment by bmwiedemann** [comment]
looks OK, but would like to test-push it to ci.suse.de

**Comment by kbaikov** [comment]
Removed the inline comments after the parameters since they would appear in string parameters in jenkins.

Did the test-push with @bmwiedemann . Resolved the hdd space problem.
The job looks more or less fine now.
Perhaps ready to merge?

**Pull request merged**